app/models/company.rb

```diff
@@ -125,7 +127,7 @@ def missing_region?
 
   def approved_applications_from_members
     # Returns ActiveRecord Relation
-    shf_applications.accepted.includes(:user)
+    shf_applications.accepted.joins(:user)
```

Comment:
There is already a method that returns current members for a company: `current_members`  That should be used to keep things DRY.  If the method for current members for a company isn't correct, then let's correct that.  (I'm not sure `current_members` is returning members that meet all current requirements.